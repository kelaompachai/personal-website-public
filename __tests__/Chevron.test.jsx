import React from 'react';
import { render } from '@testing-library/react';
import Chevron from '../src/view/Chevron';

describe('Chevron renders properly', () => {
  it('points right', () => {
    expect(render(<Chevron direction="right" />).baseElement).toMatchSnapshot();
  });

  it('points left', () => {
    expect(render(<Chevron direction="left" />).baseElement).toMatchSnapshot();
  });
});
