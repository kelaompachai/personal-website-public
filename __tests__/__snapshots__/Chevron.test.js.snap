// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chevron renders properly points left 1`] = `
<body>
  <div>
    <svg
      class="size-1/2 fill-well-read-500"
      viewBox="0 0 16 16"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M15 8a.5.5 0 0 0-.5-.5H2.707l3.147-3.146a.5.5 0 1 0-.708-.708l-4 4a.5.5 0 0 0 0 .708l4 4a.5.5 0 0 0 .708-.708L2.707 8.5H14.5A.5.5 0 0 0 15 8"
        fill-rule="evenodd"
      />
    </svg>
  </div>
</body>
`;

exports[`Chevron renders properly points right 1`] = `
<body>
  <div>
    <svg
      class="size-1/2 fill-well-read-500"
      viewBox="0 0 16 16"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M1 8a.5.5 0 0 1 .5-.5h11.793l-3.147-3.146a.5.5 0 0 1 .708-.708l4 4a.5.5 0 0 1 0 .708l-4 4a.5.5 0 0 1-.708-.708L13.293 8.5H1.5A.5.5 0 0 1 1 8"
        fill-rule="evenodd"
      />
    </svg>
  </div>
</body>
`;
